rasa_core_sdk/__init__.py

```diff
@@ -151,7 +151,23 @@ def name(self):
 
     def run(self, dispatcher, tracker, domain):
         # type: (CollectingDispatcher, Tracker, Dict[Text, Any]) -> List[dict]
-        """Execute the side effects of this action."""
+        """
+        Execute the side effects of this action.
+        Args:
+            dispatcher (CollectingDispatcher): the dispatcher which is used to
+                send messages back to the user. Use ``dipatcher.utter_message()``
+                or any other :class:`rasa_core_sdk.executor.CollectingDispatcher`
+                method.
+            tracker (Tracker): the state tracker for the current
+                user. You can access slot values using
+                ``tracker.get_slot(slot_name)``, the most recent user message
+                is ``tracker.latest_message.text`` and any other
+                :class:`rasa_core_sdk.Tracker` property.
+            domain (Dict[Text, Any]): the bot's domain
+        Returns:
+            Dict[Event]: A dictionary of :class:`rasa_core_sdk.events.Event`
```

Comment:
@ruiconti sorry, what vova meant was a list of dicts, like in your core PR:
```suggestion
            List[dict]: A dictionary of :class:`rasa_core_sdk.events.Event`
```